### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by piero-vic**
# Initialize project with components
- Initialize React App.
- Add redux, router and setup linter.
- Setup routes.
- Create placeholders for the list and the Categories section.
- Create redux file structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
```
Comment (dasileker) [inline README.md:15]: - [ ] Don't make this file the last file you will edit, please add details.

File: src/components/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import styles from './BooksList.module.css';
+
+function BooksList() {
+  return (
+    <div className={styles.container}>
+      <ul>
```
Comment (dasileker) [inline src/components/BooksList.js:6]: - [ ] The bookList should take the books as props to render the Author and Title.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @piero-vic,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Piero, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**